src/utils/emailUtil.ts

```diff
@@ -0,0 +1,87 @@
+import logger from './loggerUtil';
+import {SentMessageInfo} from 'nodemailer';
+
+require('dotenv').config({path: '../.env'});
+
+const nodemailer = require('nodemailer');
+const crypto = require('crypto');
+const {google} = require('googleapis');
+const OAuth2 = google.auth.OAuth2;
+
+const transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    type: 'OAuth2',
+    user: process.env.EMAIL_USER,
+    clientId: process.env.EMAIL_CLIENT_ID,
+    clientSecret: process.env.EMAIL_CLIENT_SECRET,
+    refreshToken: process.env.EMAIL_CLIENT_REFRESH,
+  },
+});
+
+const oAuth2Client = new OAuth2(process.env.EMAIL_CLIENT_ID, process.env.EMAIL_CLIENT_SECRET, 'https://developers.google.com/oauthplayground');
+oAuth2Client.setCredentials({
+  refresh_token: process.env.EMAIL_CLIENT_REFRESH,
+});
+
+const sendMail = (toAddress: string, subject: string, text: string, htmlText: string) => {
+  const mailOptions = {
+    from: process.env.EMAIL_USER,
+    to: toAddress,
+    subject: subject,
+    text: text,
+    html: htmlText,
+    auth: {
+      accessToken: oAuth2Client.getAccessToken(),
+    },
+  };
+
+  transporter.sendMail(mailOptions, function(error: Error | null, info: SentMessageInfo) {
+    if (error) {
+      logger.error(error);
```

Comment:
Can we get an embed here that just tells the user there was a fatal error in processing this command? Something like "Yo, having trouble, sorry!". Or this could be the caller's responsibility, but your choice